http/graphql-telemetry/src/test/java/io/quarkus/ts/http/graphql/Utils.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.ts.http.graphql;
+
+import static io.restassured.RestAssured.given;
+
+import javax.json.Json;
+
+import io.restassured.response.Response;
+
+public class Utils {
```

Comment:
Is there any reason why this is used for now rather than SmallRye GraphQL Client?

Reply:
Yes, this is easier to reproduce with curl or similar instruments.

And we can be sure, that any problems are due to bugs on server and not on client

Reply:
Is there any plan to test the client too? According to list of extensions, the client is going to be part of product for Elektra.

Reply:
I haven't seen such plan. But I can add support for client as part of this MR

Reply:
GraphQL client is preview on upstream for now.

I'm gonna merge this for now as I don't presume this is something that is going to be supported. But still, can you ask what is the plan for client with Elektra on JIRA, @fedinskiy ?